types/got/index.d.ts

```diff
@@ -59,13 +64,14 @@ declare class StdError extends Error {
 declare const got: got.GotFn &
     Record<'get' | 'post' | 'put' | 'patch' | 'head' | 'delete', got.GotFn> &
     {
-        stream: got.GotStreamFn & Record<'get' | 'post' | 'put' | 'patch' | 'head' | 'delete', got.GotStreamFn>
-        RequestError: typeof RequestError
-        ReadError: typeof ReadError
-        ParseError: typeof ParseError
-        HTTPError: typeof HTTPError
-        MaxRedirectsError: typeof MaxRedirectsError
-        UnsupportedProtocolError: typeof UnsupportedProtocolError
+        stream: got.GotStreamFn & Record<'get' | 'post' | 'put' | 'patch' | 'head' | 'delete', got.GotStreamFn>;
```

Comment:
There I just add semicolon `;` to the end of each line. If this change is redundant I can revert it.